### PR TITLE
Added server restart service file. Changed filepaths to absolute.

### PR DESCRIPTION
**Pull request by SzymonFed**

### CODE REVIEW

File: gateway/gateway.cpp [hunk 1]
```diff
@@ -134,6 +138,14 @@ int main(void)
...
     });
 
+    /* Crashes server for testing purposes ;d */
+    CROW_ROUTE(app, "/crash")
+    ([&](const crow::request &req) -> crow::response {
+        pthread_rwlock_rdlock(&q_rwlock);
+        raise(SIGSEGV);
+        pthread_rwlock_unlock(&q_rwlock);
+        return std::string("Hello world\n");
+    });
```
Comment (kuba8686) [inline gateway/gateway.cpp:148]: :)

Comment (Siwy160) [reply gateway/gateway.cpp:148]: I'm not sure if we want such thing.

This should not be merged to master branch.

Comment (Siwy160) [reply gateway/gateway.cpp:148]: Keep it on your separate branch. Remove from here.

Comment (SzymonFed) [reply gateway/gateway.cpp:148]: Forgot to remove that part before push ;d

**Pull request merged**